cypress/page/login-page.ts

```diff
@@ -0,0 +1,25 @@
+class LoginPage {
+    private loginURL: string;
+    private userNameField: string;
+    private passwordField: string;
+    private loginButton: string;
+  
+    constructor() {
+      this.loginURL = "https://www.saucedemo.com/";
+      this.userNameField = ".login-box > form > div > input#user-name";
+      this.passwordField = ".login-box > form > div > input#password";
+      this.loginButton = "input[type='submit']";
```

Comment:
Is there a way to improve these selectors?

Reply:
Those locators were updated by ID